src/main/java/org/aion/wallet/ui/components/HistoryController.java

```diff
@@ -231,11 +229,10 @@ private void redirect(final TableView<?> table) {
         private TxRow(final String requestingAddress, final TransactionDTO dto) {
             transaction = dto;
             final AccountDTO fromAccount = blockchainConnector.getAccount(dto.getFrom());
-            final AccountDTO toAccount = blockchainConnector.getAccount(dto.getTo());
             final String balance = BalanceUtils.formatBalance(dto.getValue());
             boolean isFromTx = AddressUtils.equals(requestingAddress, fromAccount.getPublicAddress());
             this.type = new SimpleStringProperty(isFromTx ? TO : FROM);
-            this.name = new SimpleStringProperty(isFromTx ? toAccount.getName() : fromAccount.getName());
+            this.date = new SimpleStringProperty(new SimpleDateFormat("yyyy/MM/dd - HH.mm.ss").format(new Date(dto.getTimeStamp() * 1000)));
```

Comment:
`SimpleDateFormat` can be `static final` field of `HistoryController`

Reply:
:+1: